app/views/customer_service/index.html.erb

```diff
@@ -0,0 +1,332 @@
+            </tr>
+          </tbody>
+        </table>
+        Contact for:
+        <ul>
+        	<li>Website Navigation Help</li>
+          <li>Account Issues</li>
+          <li>Technical Help</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="usa-grid-full customer-service-section" id="svc-help">
+      <div class="usa-width-one-fourth">
+        <%= image_tag("svc-help.png") %>
+        <h4>Service Branch Help Desks</h4>
+      </div>
+      <div class="usa-width-three-fourths">
+        <%- @contacts.each do |contact| -%>
+          <% if contact.custsvc_org %>
+            <%= contact.custsvc_org %>
+            <table>
+              <tbody>
+                <% if contact.custsvc_dsn %>
+                  <tr>
+                    <th scope="row">Phone</th>
+                    <td>DSN <%= contact.custsvc_dsn %></td>
+                  </tr>
+                <% end %>
+                <% if contact.custsvc_tel_comm %>
+                  <tr>
+                    <% if contact.custsvc_dsn %>
+                      <td></td>
+                    <% else %>
+                      <th scope="row">Phone</th>
+                    <% end %>
+                    <td>Commercial <%= link_to contact.custsvc_tel_comm, "tel:" + contact.custsvc_tel_comm %></td>
+                  </tr>
+                <% end %>
+                <% if contact.custsvc_tel_tollfree %>
+                  <tr>
+                    <% if contact.custsvc_dsn || contact.custsvc_tel_comm %>
+                      <td></td>
+                    <% else %>
+                      <th scope="row">Phone</th>
+                    <% end %>
+                    <td>Toll-Free <%= link_to contact.custsvc_tel_tollfree, "tel:" + contact.custsvc_tel_tollfree %></td>
+                  </tr>
+                <% end %>
+                <% if contact.custsvc_email %>
+                  <tr>
+                    <th scope="row">Email</th>
+                    <td><%= link_to contact.custsvc_email.sub(',', '<br').html_safe, "mailto:" + contact.custsvc_email %></td>
+                  </tr>
+                <% end %>
+                <% if contact.custsvc_hours %>
+                  <tr>
+                    <th scope="row">Hours</th>
+                    <td><%= contact.custsvc_hours %></td>
+                  </tr>
+                <% end %>
+                <% if contact.custsvc_url %>
+                  <tr>
+                    <th scope="row">Online</th>
+                    <td><%= link_to contact.custsvc_url, contact.custsvc_url, class: "usa-external_link" %></td>
+                  </tr>
+                <% end %>
+                <% if contact.custsvc_facebook_url %>
+                  <tr>
+                    <td></td>
+                    <td><%= link_to image_tag("FB-FindUsonFacebook-online-512.png", :alt => "Find Us On Facebook", :width => "98"), contact.custsvc_facebook_url %></td>
```

Comment:
Not only rename Facebook icon, `:alt => 'Find us on Facebook'` is the old Ruby syntax. ✨